roles/neutron-common/templates/etc/serverspec/neutron_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+describe file('/etc/neutron/rootwrap.d') do
+  it { should be_owned_by 'root' }
+  it { should be_grouped_into 'root' }
+  it { should be_writable.by('owner') }
+end    
+
+describe file('/opt/openstack/current/neutron/etc/neutron/rootwrap.d') do
+  it { should be_owned_by 'root' }
+  it { should be_grouped_into 'root' }
+  it { should be_writable.by('owner') }
+end  
+
+files = ['bin', 'sbin']
+files.each do |file|
+  describe file("/#{file}") do
+    it { should be_owned_by 'root' }
+    it { should be_grouped_into 'root' }
+    it { should be_writable.by('owner') }
+  end 
+end 
+
+files = ['bin', 'sbin']
+files.each do |file|
+  describe file("/usr/#{file}") do
+    it { should be_owned_by 'root' }
+    it { should be_grouped_into 'root' }
+    it { should be_writable.by('owner') }
+  end 
+end 
+
+files = {'api-paste.ini' => 640, 'dhcp_agent.ini'=> 640, 'l3_agent.ini'=>640, 'metadata_agent.ini'=>640, 'policy.json' => 644 ,'neutron.conf'=> 640, "rootwrap.conf" => 640}
+files.each do |file, mode|
+  describe file("/etc/neutron/#{file}") do
+    it { should be_mode mode }
+    it { should be_owned_by 'neutron' }
+    it { should be_grouped_into 'neutron' }
+  end
+end
+
+files = ['neutron-dhcp-agent.log' , 'neutron-l3-agent.log' , 'neutron-linuxbridge-agent.log' , 'neutron-metadata-agent.log']
+files.each do |file|
+  describe file("/var/log/neutron/#{file}") do
+    it { should be_owned_by 'neutron' }
+    it { should be_grouped_into 'adm' }
+  end
+end   
+
+describe file('/etc/logrotate.d/neutron') do
+  file_contents = [ '# Generated by Ansible.',
+           '# Local modifications will be overwritten.',
+         '',
+         '/var/log/neutron/*.log',
+         '{',
+         '  daily',
+         '  missingok',
+         '  rotate 7',
+         '  compress',
+         '}']
+
+  file_contents.each do |file_line|
+    it { should contain file_line}
+  end
+end
+
+describe file('/etc/neutron/neutron.conf') do
+  it {should contain 'debug = True'}
```

Comment:
The value of this should be derived from the variable value for `neutron.logging.debug`.